README.md

```diff
@@ -7,10 +7,19 @@
 
 This repository contains the latest version of Human-GEM, a human genome-scale metabolic model.
 
-### Citation
+### Cite us:
+
+If you use Mouse1, Rat1, Zebrafish1, Fruitfly1 or Worm1 in your research, please cite:   
+
+ > H. Wang, J. L. Robinson, P. Kocabaş, J. Gustafsson, M. Anton, P.-E. Cholley, et al. Genome-scale metabolic network reconstruction of model animals as a platform for translational research. _PNAS_ 118, e2102344118 (2021). [doi.org/10.1073/pnas.2102344118](https://doi.org/10.1073/pnas.2102344118)
+
+If you use Human1 in your research, please cite:  
 
  > J. L. Robinson, P. Kocabaş, H. Wang, P.-E. Cholley, et al. An atlas of human metabolism. _Sci. Signal._ 13, eaaz1482 (2020). [doi:10.1126/scisignal.aaz1482](https://doi.org/10.1126/scisignal.aaz1482)
 
+Starting from Human-GEM v1.5.0, all the releases are also archived in [Zenodo](https://doi.org/10.5281/zenodo.4099692) from which specific version can be cited if used.
```

Comment:
The ordering here is based on the most recent work. However, this being the repository for Human-GEM, I expect most visitors to be interested in reading about or citing Human1, so I am tempted to place that first - simply reordering without changing the text would be fine by me.

Reply:
sounds good to me